pick.c

```diff
@@ -800,11 +801,12 @@ print_choices(int offset, int selection)
 enum key
 get_key(char *buf, size_t size, size_t *nread)
 {
-#define	KEY(k, s)	{ k, s, sizeof(s) - 1 }
+#define	CAP(k, s)	{ k,	s,	NULL }
+#define	KEY(k, s)	{ k,	NULL,	s }
 	static struct {
 		enum key	 key;
-		const char	*s;
-		size_t		 length;
+		char		*cap;
```

Comment:
I was wondering why this wasn't `const char` but then I saw `tigetstr` takes a `char *`. I'm not quite sure why though...